unittests/platformagnostictests.py

```diff
@@ -342,6 +342,21 @@ def test_format_empty_file(self) -> None:
         for code in ('', '\n'):
             formatted = formatter.format(code, Path())
             self.assertEqual('\n', formatted)
+    
+    def test_format_indent_comment_in_brackets(self) -> None:
+        """Issue #13508"""
```

Comment:
In mesa we found these sort of comments be not helpful for a lot of reasons. I don't mind having a link to the issue, but having at least a short text summary of what's being checked and fixed would be good to.